dispatcher/frontend-ui/src/components/ScheduleEditor.vue

```diff
@@ -329,7 +329,9 @@
           if (field.type == "string-enum") {
             component = "b-form-select";
             options = field.choices.map(function (option) { return {text: option, value: option}; });
-            options.push({text: "Not set", value: undefined});
+            if (field.required != true) {
+              options.push({text: "Not set", value: undefined});
```

Comment:
Please make sure to check zimit-frontend about this. I'm think this part is duplicated so it might need to be backported there as well

Reply:
https://github.com/openzim/zimit-frontend/pull/38